src/org/integr8ly/GitHubUtils.groovy

```diff
@@ -17,22 +17,20 @@ import org.kohsuke.github.PagedIterable
  * @param endpoint - Github endpoint to send the request to (https://developer.github.com/v3/apps/available-endpoints/)
  * @param httpMethod - HTTP method to be used for the request
  * @param requestBody - HTTP request body
- * @param ghApiTokenID - The credentials id where the github api token is stored in Jenkins
+ * @param credentialsID - The credentials id where the github user/password is stored in Jenkins
  * @param customHeaders - Additional headers to be used for the request
  * @param validResponseCodes - Valid HTTP response codes
  * @returns HTTP response
  */
-def ghApiRequest(String endpoint, String httpMethod = 'GET', String requestBody = null, String ghApiTokenID, customHeaders = [], validResponseCodes = '100:399') {
+def ghApiRequest(String endpoint, String httpMethod = 'GET', String requestBody = null, String credentialsID, validResponseCodes = '100:399') {
```

Comment:
I think you could have left "customHeaders" as it was tbh. Was doing no harm and is meant to allow you to easily re-use this method in the case where you need to set custom headers.